script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_2_1.txt

```diff
@@ -50,7 +50,7 @@ It's that, you know. I just wanted to make sure you weren't feeling blue."
 "It wasn't an interesting enough show to get up early for. Disappointing, very disappointing."
 "I'm sorry I couldn't live up to your expectations. You've got too much free time."
 <Page14>
-"It's because of that free time that I like gossip so much. You've been living somewhere else since elementary school, right?#I don't know what the reason was, but it's been seven years since then. Why did your father decide to bring his disowned son back now?"
+"It's because of that free time that I like gossip so much. You've been living somewhere else since elementary school, right? # I don't know what the reason was, but it's been seven years since then. Why did your father decide to bring his disowned son back now?"
```

Comment:
There should probably be a ` ` on only one side of the `#`, otherwise these two sentences will end up double-spaced from each other. Same thing happens in the file before, but I think that for separating the quoted text it's not as bad to have the extra width